src/provisioner/provisioner.py

```diff
@@ -274,6 +314,27 @@ def cluster_bootstrap(cortx_conf_url: str, force_override: bool = False):
             cortx_conf, node_id, apply_phase, ProvisionerStatus.SUCCESS.value)
         Log.info(f"Finished cluster bootstrap on {node_id}:{node_name}")
 
+    @staticmethod
+    def prepare_delta(_conf_idx: str, _tmp_idx: str):
```

Comment:
If this is internally called function that it must start with _
Can we make function / variable names more meaningful and add description. For example, diff or prepare_diff is more meaningful here with an obvious expectation that it would return delta / diff

Signature should be more like to make it more easy to read. 
**save_diff(idx1, idx2, diff_idx)**

Reply:
done